### PR TITLE
Create SplitStringIntoArrayOfWords.java

### LINKED ISSUE #4292: Write a Java program to split string into array of words
### Description

Write a Java program to split string into array of words

```
Input  : "hello world"
Output : ["hello", "world"]
```
<details>
<summary>How to contribute</summary>

- Save the solution in `SplitStringIntoArrayOfWords.java` file
- Add `SplitStringIntoArrayOfWords.java` file in `program/split-string-into-array-of-words` folder

</details>

### PR DESCRIPTION
**Pull request by dineshlalam15**
This is a short and efficient java code to split the string into an array of words.
Closes codinasion/codinasion#4292

### CODE REVIEW

File: program/split-string-into-array-of-words/SplitStringIntoArrayOfWords.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import java.util.Scanner;
+
+public class SplitStringIntoArrayOfWords {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Enter the String: ");
+        String str = sc.nextLine();
+        System.out.println(Arrays.toString(str.split(" ")));
+    }
+}
```
Comment (anandfresh) [inline program/split-string-into-array-of-words/SplitStringIntoArrayOfWords.java:11]: It should return a string array

**Review by anandfresh — CHANGES_REQUESTED** [review]

File: program/split-string-into-array-of-words/SplitStringIntoArrayOfWords.java (hunk 1 above)
Comment (dineshlalam15) [reply program/split-string-into-array-of-words/SplitStringIntoArrayOfWords.java:11]: String.str(" "); gives a STRING ARRAY only! 
For better understanding I'll edit the code again. 
Thank you sir.

**Review by anandfresh — CHANGES_REQUESTED** [review]
Can you please assign the issue (https://github.com/codinasion/program/issues/5646 ) to your name .
Just type '!assign' in the issue comment , it will assign you to the issue

**Comment by dineshlalam15** [comment]
!assign

**Comment by harshraj8843** [comment]
Hey @dineshlalam15, `!assign` command is only for `good first issue` !!!

Please choose another issue.

Thanks for your interest in contributing to this project.

**Comment by dineshlalam15** [comment]
@anandfresh I'm a newbie to GitHub and contributions. It would be great if you assist me with "How to assign an issue".
BTW I created a new issue with my name and commented, !assign. 
https://github.com/codinasion/codinasion/issues/4305

**Review by anandfresh — APPROVED** [review]

**Comment by harshraj8843** [comment]
> @anandfresh I'm a newbie to GitHub and contributions. It would be great if you assist me with "How to assign an issue". BTW I created a new issue with my name and commented, !assign. codinasion/codinasion#4305

Hey @dineshlalam15 

You can comment `!assign` on any `good first issue` you want to contribute to, to be auto-assigned.

For this PR case, you should first comment `!assign` on Issue codinasion/codinasion#4292 and then create a PR. This helps in preventing conflict for PR(s) for the same issue by different contributors 👍🏻 

> **Note** Please do not create a new issue with just `!assign`

**Comment by dineshlalam15** [comment]
Much helpful! 
Thank you so much

**Pull request merged**